tests/nlu/classifiers/test_diet_classifier.py

```diff
@@ -49,6 +52,120 @@
 from rasa.utils.tensorflow.model_data_utils import FeatureArray
 
 
+@pytest.fixture()
+def default_diet_resource() -> Resource:
+    return Resource("DIET")
+
+
+@pytest.fixture()
+def create_diet(
+    default_model_storage: ModelStorage,
+    default_execution_context: ExecutionContext,
+    default_diet_resource: Resource,
+) -> Callable[..., DIETClassifier]:
+    def inner(
+        config: Dict[Text, Any], load: bool = False, finetune: bool = False
+    ) -> DIETClassifier:
+        if load:
+            constructor = DIETClassifier.load
+        else:
+            constructor = DIETClassifier.create
+
+        default_execution_context.is_finetuning = finetune
+        return constructor(
+            config={**DIETClassifier.get_default_config(), **config},
+            model_storage=default_model_storage,
+            execution_context=default_execution_context,
+            resource=default_diet_resource,
+        )
+
+    return inner
+
+
+@pytest.fixture()
+def create_train_load_and_process_diet(
+    nlu_data_path: Text,
+    create_diet: Callable[..., DIETClassifier],
+    default_model_storage: ModelStorage,
```

Comment:
Shouldn't we be able to drop e.g. `default_model_storage` here, as we already have `create_diet`?

Reply:
(same below)

Reply:
good catch